app/src/main/assets/verses/type0/es/type0.json

```diff
@@ -8,31 +8,31 @@
     "9": "Dudoso",
     "10": "Afrontar las dificultades",
     "11": "Sentir celos",
-    "12": "Para los hijos desobedientes",
+    "12": "Para niños desobedientes",
     "13": "Para el empleo",
     "14": "Para el amor entre el marido y la esposa",
     "16": "Para aumentar los medios de subsistencia",
     "17": "Dolor de cabeza (Cefaleas)",
     "18": "Soledad",
     "19": "Necesidad de un amigo",
-    "20": "No hay esperanza de tener hijos",
+    "20": "No hay esperanza para los niños",
     "21": "No apreciado",
-    "22": "Dolor hepático (Hígado)",
-    "23": "Tristeza",
+    "22": "Dolor hepático",
+    "23": "Triste",
     "24": "Miedo",
     "25": "Pedir perdón",
     "27": "Pecado",
     "31": "Para quitar los problemas de la cabeza",
     "32": "Malos tratos",
-    "33": "Confía en Alá a la hora de tomar decisiones",
+    "33": "Confía en Alá cuando tomes decisiones",
     "34": "Inquietudes en el corazón",
     "35": "Problemas con la vista",
     "36": "Problemas de estómago",
-    "37": "Cuando se pierde algo",
+    "37": "Cuando algo se ha perdido",
     "4": "La ira",
     "2": "Contra la brujería",
-    "15": "Para aprobar los exámenes",
+    "15": "Para tener éxito en los exámenes",
     "30": "Eliminar la ira",
-    "28": "Dolor estomacal (estómago)",
-    "29": "En las dificultades"
+    "28": "Dolor estomacal",
+    "29": "En los conflictos"
```

Comment:
Good afternoon:

About the chain: 

Dolor estomacal (28) in Spanish means A pain in the stomach. But I put a simpler correction in case the word estomacal causes mistakes. 

And about chain 29. I did the same. Since it can refer to problems, fight, etc...

A hug 

Pd: If you have any questions, do not hesitate to let me know.  :)

Reply:
I specifically have question about the key "29".

Is "conflictos" right word for "struggling"?

Reply:
Good afternoon @faisalcodes

It's not the same thing, well or yes. Since for conflicts there are struggles (fight).

But upon closer look I realized in that you were referring to Struggle (fight). And I already made the correction. 

Since conflicts, worries, inner struggles (fight), and more words that can be associated can be encompassed.

So I made the correction. 

Anything else feel free to let me know. Thanks for the note :)

Thanks for your time.

Reply:
Good afternoon!
You have been contributing a lot and it is appreciated by the community.

Thankyou for the correction. I wanted to make sure everything is as good as possible.

Reply:
Good afternoon @faisalcodes

The appreciation is mutual :)

Too bad some words in English. They can have multiple ways of saying the same thing in Spanish. And sometimes looking for the ideal is complicated.

Since maybe I put a synonym and it is not the exact word you are looking for. So anything I see that I should correct or change. Feel free to let me know, I will try to: Darle una vuelta de tuerca (To give it a twist) to better adapt the translation.

And if I mess up, don't hesitate to let me know. My neuron can get away from me XD

Best regards

Reply:
So far everything has been fine. I only point out what it feels like different. To make it clear, I use Google translate to verify and it is not always correct, so I may be incorrect to point out sometimes.

The main motive of translations are to make the people easier to understand the purpose. That's why, translations should not necessarily be literal.

Reply:
Good afternoon @faisalcodes

Well, if you see something wrong, please let me know. Because I may miss something. Although I try to put all my neuron in it.

By the way, if you allow me. I wanted to ask you about some Weblate strings. Since I do not know why it gives error.

The strings are:

https://hosted.weblate.org/translate/QuranApp/quranic-topics/es/?q=has:dismissed-check In this string is https://es.m.wikipedia.org/wiki/Pecado ( https://en.m.wikipedia.org/wiki/Sin ). And as it is past tense, since it ends in -ed , it would be pequé . But I think it would be better to leave it as Sin, in order to refer to all verb tenses. I don't know what you think?

This https://hosted.weblate.org/translate/QuranApp/quranic-topics/es/?q=has%3Adismissed-check&offset=2 I don't know why it gives error. Since there is no more translation. It could be shortened but it would not sound good.

And this https://hosted.weblate.org/translate/QuranApp/quranic-etiquette/es/?q=has:dismissed-check I don't see the flaw anywhere either.

Then there is https://hosted.weblate.org/translate/QuranApp/quranic-etiquette/es/?q=has:check . Here the only thing I can think of is to leave the word compulsion (in Spanish it has an accent on the o). That would be https://es.m.wikipedia.org/wiki/Compulsi%C3%B3n . But if I didn't misunderstand the expression, it refers to obligatory, coercion. Or do you mean addictive or similar...? It is the latter if I messed up when translating. 

A hug and sorry for my mistakes.

Reply:
Thank you for the deep analysis. They are not errors so please don't mind them. The checks say the strings are not translated in other languages. Once they get translated in other languages, you will find the errors to be disappeared. I regularly keep checking about errors of this kind and resolve it myself.

Reply:
Good afternoon @faisalcodes

No need to say thank you.

But I missed that they were marked as failures.

Power, you can say a sentence in several ways.

For example:

Aguantar la ira o la rabia
Contener la ira  o la rabia
Reprimir la ira o la rabia 

Etc...

That's why I thought that this could be where the failures could come from. 

The only thing is that if you want, you could look for another way to express the same thing.

If I can help you know, don't hesitate to tell me ;)

Best regards